joomla/joomla-cms
# [5.2] Unable to store null value on update row #39363

**Pull Request for Issue** #39363

**Summary of Changes** 
Replace isset() with array_key_exists() which also returns true when property has null as value.

**Testing Instruction** 
Same as mention on issue `https://github.com/joomla/joomla-cms/issues/39363`

Result before appling this PR
On update column that have updated value null not change in database

Expected result after this PR
Column that have updated value null will also change in database




### Link to documentations
 No documentation changes for docs.joomla.org needed
 No documentation changes for manual.joomla.org needed
